README.md

```diff
@@ -255,4 +285,25 @@ Using scaled threading model. 24 processors detected, 24 threads used to drive t
 ===== DaCapo 9.12-MR1 lusearch PASSED in 822 msec =====
 ```
 
-**Note:** Pass `-XX:+UseThirdPartyHeap` as java command line arguments to enable MMTk.
+### MMTk options
+
+MMTk has many options defined in https://github.com/mmtk/mmtk-core/blob/master/src/util/options.rs
+
+You can use environment variables started with `MMTK_` to set those options.  For example,
+`export MMTK_THREADS=1` will set the number of GC worker threads to one.  Follow the link above for
+more details.
+
+You can also set those options via command line arguments.
+
+-   `-XX:ParallelGCThreads=n` (where `n` is a number) sets the number of GC worker threads.
```

Comment:
Could you clarify that this option actually changes both the number of STW/parallel GC threads and the concurrent GC threads for MMTk?

For OpenJDK GCs, `-XX:ParallelGCThreads=n` only changes the number of STW threads IIRC.

Reply:
Currently concurrent GC has not been merged into mmtk-core. So when Wenyu merges his branch, we can introduce the same interface (no of concurrent GC threads) to make it similar to the OpenJDK interface

Reply:
I added clarification about `-XX:ConcGCThreads`.  Because we haven't merged concurrent plans, yet, I just mentioned that `ConcGCThreads` is ignored when using MMTk.  When we merge concurrent plans, we can update this line of documentation, and maybe also adjust the MMTk options (such as introducing `Options::concurrent_threads`, etc.) that matches the OpenJDK options.